packages/front-end/components/Experiment/SinglePage.tsx

```diff
@@ -51,6 +54,37 @@ function getColWidth(v: number) {
   return 3;
 }
 
+function drawMetricRow(
+  m: string,
+  metric: MetricInterface,
+  experiment: ExperimentInterfaceStringDates
+) {
+  const newMetric = structuredClone(metric) as MetricInterface;
+  const override = applyMetricOverrides(newMetric, experiment);
+  return (
+    <div key={m} className="ml-2">
+      <span className="mr-1">-</span>
+      <Link href={`/metric/${m}`}>
+        <a className="mr-2 font-weight-bold">{newMetric?.name}</a>
+      </Link>
+      {newMetric && (
+        <div className="ml-4 small">
+          <label className="text-muted">Conversion Window:</label>{" "}
+          <span>
+            {newMetric.conversionDelayHours
+              ? newMetric.conversionDelayHours + " to "
+              : ""}
+            {(newMetric.conversionDelayHours || 0) +
+              (newMetric.conversionWindowHours ||
+                getDefaultConversionWindowHours())}{" "}
+            hours {override && <span className="font-italic">(override)</span>}
+          </span>
+        </div>
+      )}
+    </div>
```

Comment:
What do you think about using tables as an alternate layout here?  Makes it more compact

![image](https://user-images.githubusercontent.com/1087514/202231677-d54cc7f6-f24b-4bde-876a-f97ac8672f56.png)